src/main/java/io/stargate/sgv2/jsonapi/util/naming/CollectionNamingRule.java

```diff
@@ -0,0 +1,7 @@
+package io.stargate.sgv2.jsonapi.util.naming;
+
+public class CollectionNamingRule extends SchemaObjectNamingRule {
+  public CollectionNamingRule(String name) {
```

Comment:
Do we need to pass name here, or can it be just defined here as:

   super("collection")

(or whatever constant we use)?